src/constants/dimension.js

```diff
@@ -64,6 +64,12 @@ export const dimConf = {
         objectName: 'pi',
         itemType: 'PROGRAM_INDICATOR',
     },
+    calculation: {
+        value: 'expressionDimensionItems',
+        dimensionName: 'dx',
+        objectName: 'ed', // Created by Bjorn, don't seem to be in use when the map is saved
```

Comment:
@turban could you expand on your comment a bit (or explain it in the response here in gh?